{"name": "test_sample_ui_test_case", "status": "broken", "statusDetails": {"message": "AttributeError: 'WebDriver' object has no attribute 'driver'", "trace": "self = <TestCases.testsuites.test_sample.TestSample1 object at 0x0000024A52364090>\nuiTestData = {'discharge_date': '01/18/2023', 'facility': {'facility_id': 'FA03', 'facility_name': 'Teamfacility'}, 'member': {'DOB...TN23', 'Member_id': 'TN22', ...}, 'password': '80f7d73c5456-430da38f0b7de6131f170c57755653164ab2a353a1cc138d12b9', ...}\nconfigLogger = <RootLogger root (INFO)>\n\n    def test_sample_ui_test_case(self,uiTestData, configLogger):\n        driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()))\n        driver.maximize_window()\n        login(uiTestData=uiTestData,driver=driver)\n>       login_web(uiTestData=uiTestData,driver=driver)\n\nTestCases\\testsuites\\test_sample.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nCommonClass\\web_utils\\CommonUiActions.py:45: in login_web\n    Member.create_Member(driver)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ff74abfe1390259d33dd20d0fa91a272\")>\n\n    def create_Member(self):\n>       self.driver.find_element(By.XPATH, self.member_search_id).click()\nE       AttributeError: 'WebDriver' object has no attribute 'driver'\n\nCommonClass\\web_utils\\pages\\member.py:16: AttributeError"}, "steps": [{"name": "Log (INFO) ====== WebDriver manager ======", "status": "passed", "start": 1674128196702, "stop": 1674128196702}, {"name": "Log (INFO) Get LATEST chromedriver version for google-chrome 108.0.5359", "status": "passed", "start": 1674128199677, "stop": 1674128199677}, {"name": "Log (INFO) Driver [D:\\Users\\pvelu\\.wdm\\drivers\\chromedriver\\win32\\108.0.5359\\chromedriver.exe] found in cache", "status": "passed", "start": 1674128201441, "stop": 1674128201441}], "attachments": [{"name": "log", "source": "a90b6bb9-79a1-4fe6-b0a3-9103246f6227-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "df2114f8-9702-478b-a4ae-b3fb54b72301-attachment.txt", "type": "text/plain"}], "start": 1674128196702, "stop": 1674128224295, "uuid": "81213463-07f6-452c-adb2-b4d0ab6ee2b0", "historyId": "1971dd38761290445127c4af03b141fd", "testCaseId": "2186558a34288d22382cd9d3e416f696", "fullName": "TestCases.testsuites.test_sample.TestSample1#test_sample_ui_test_case", "labels": [{"name": "parentSuite", "value": "TestCases.testsuites"}, {"name": "suite", "value": "test_sample"}, {"name": "subSuite", "value": "TestSample1"}, {"name": "host", "value": "WSAMZN-7K50OVEF"}, {"name": "thread", "value": "5344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.testsuites.test_sample"}]}